### PR TITLE
[ENG-798] - Add daily action to update GraphQL TS schema

### PR DESCRIPTION
**Pull request by floreks**
## Summary

Added a scheduled cron action that runs every day at 05:00 (UTC?), runs the graphql codegen and commits it back to the repo.

## Checklist

- [ ] If required, I have updated the Plural documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have added a meaningful title and summary to convey the impact of this PR to a user.
- [ ] I have added relevant labels to this PR to help with categorization for release notes.

### CODE REVIEW

File: .github/workflows/daily.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.15.0
+      - name: 'Update Schema'
+        run: |
+          yarn install --immutable
+          yarn graphql:codegen
+          git add .
+          git commit -m "Update GraphQL TS Schema"
+          git push
```
Comment (floreks) [inline .github/workflows/daily.yml:31]: Will it work without using github token? @DavidSpek any idea?

**Review by michaeljguarino — APPROVED** [review]
good idea

File: .github/workflows/daily.yml (hunk 1 above)
Comment (maciaszczykm) [reply .github/workflows/daily.yml:31]: It will fail as we have branch protection rules set for `master`.

**Review by dherault — APPROVED** [review]

**Comment by floreks** [comment]
Let's merge and see if builtin token will have enough privileges to push back to the repo.

**Pull request merged**